File: docs/components_page/components/list_group/links.jl
using DashBootstrapComponents, DashHtmlComponents

list_group = html_div([
    dbc_listgroup([
        dbc_listgroupitem("Internal link", href = "/l/components/list_group"),
        dbc_listgroupitem("External link", href = "https://google.com"),
        dbc_listgroupitem("Disabled link", href = "https://google.com", disabled = true),
        dbc_listgroupitem("Button", id = "button-item", n_clicks = 0, action = true),
    ]),
    html_p(id = "counter"),
]);


callback!(app, Output("counter", "children"), Input("button-item", "n_clicks")) do n
    return "Button clicked $n times"
end;

File: docs/components_page/components/list_group/links.R
library(dashBootstrapComponents)

list_group <- div(
  list(
    dbcListGroup(
      list(
        dbcListGroupItem(
          "Internal link",
          href = "/l/components/list_group"
        ),
        dbcListGroupItem("External link", href = "https://google.com"),
        dbcListGroupItem(
          "Disabled link",
          href = "https://google.com", disabled = TRUE
        ),
        dbcListGroupItem(
          "Button",
          id = "button-item", n_clicks = 0, action = TRUE
        )
      )
    ),
    p(id = "counter")
  )
)


app$callback(
  output("counter", "children"),
  list(input("button-item", "n_clicks")),
  function(n) {
    return(sprintf("Button clicked %d times", n))
  }
)
